.gitignore

```diff
@@ -23,3 +23,6 @@ light-client-js/pkg/
 light-client-js/examples/verifier-web/node_modules/
 light-client-js/examples/verifier-web/dist/
 light-client-js/examples/verifier-web/package-lock.json
+
+# clion
+.idea
```

Comment:
This exclude is not meaningful outside of specific personal setups. Why do you want to see it included?

Reply:
Yes, this should probably go in `.git/info/exclude`, cf. http://git-scm.com/docs/gitignore.

Reply:
> This should probably go in `.git/info/exclude` (cf. http://git-scm.com/docs/gitignore).

Oh, Thanks for you, This is help me.